policyengine_us/variables/household/income/person/is_paid_hourly.py

```diff
@@ -0,0 +1,9 @@
+from policyengine_us.model_api import *
+
+
+class is_paid_hourly(Variable):
+    value_type = bool
+    entity = Person
+    label = "is paid hourly"
+    reference = "https://www.federalregister.gov/documents/2024/04/26/2024-08038/defining-and-delimiting-the-exemptions-for-executive-administrative-professional-outside-sales-and"
```

Comment:
I think this reference should go in wherever `is_paid_hourly` is used in the formula itself, we don't need a reference to back up the existence of the paid-hourly property of a person.